### PR TITLE
Create access-web-services.md

### PR DESCRIPTION
**Pull request by alabulei1**

### CODE REVIEW

**Comment by alabulei1** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/getting-started-developer/webhook-access-web-services.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+}
+```
+
+### Handle the request
+The handler annotated with `#[request_handler]` will handle the incoming HTTP request. The function first fetches the "city" query parameter, then calls `get_weather` to retrieve weather data for the specified city. 
+
+```rust
+#[request_handler]
+async fn handler(_headers: Vec<(String, String)>, _subpath: String, qry: HashMap<String, Value>, _body: Vec<u8>) {
+    ...
```
Comment (juntao) [inline docs/getting-started-developer/webhook-access-web-services.md:63]: Need to show it gets the city name and then calls get_weather

File: docs/getting-started-developer/webhook-access-web-services.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+struct Main {
+    temp_max: f64,
+    temp_min: f64,
+}
+
+#[derive(Deserialize)]
+struct Wind {
+    speed: f64,
+}
+```
```
[the remark this answers is not in the document]
Comment (alabulei1) [reply docs/getting-started-developer/webhook-access-web-services.md:161]: Added

**Comment by alabulei1** [comment]
@juntao Please review it again!

**Review by juntao — APPROVED** [review]

**Pull request merged**